src/test/java/org/ow2/proactive/catalog/service/CatalogObjectServiceTest.java

```diff
@@ -397,10 +406,11 @@ public void testUpdateObjectMetadataOnlyKindOrOnlyContentTypeOrOnlyProjectName()
                                                                                                           Optional.of("updated-kind"),
                                                                                                           Optional.of("updated-contentType"),
                                                                                                           Optional.of("updated-projectName"),
-                                                                                                          Optional.of(TAGS));
+                                                                                                          Optional.of(TAGS),
+                                                                                                          USERNAME);
 
         assertThat(catalogObjectUpdatedProjectName).isNotNull();
-        assertThat(catalogObjectUpdatedProjectName.getCommitMessage()).isEqualTo(COMMIT_MESSAGE);
+        assertThat(catalogObjectUpdatedProjectName.getCommitMessage()).isEqualTo("The project name or/and tags metadata are updated");
```

Comment:
if this commit message is used in multiple place, consider defining a constant

Reply:
OK